### DISCUSSION TITLE
Parental controls

### DISCUSSION THREAD

**Discussion by wingwang** [2Fake/devolo_plc_api#93] [Q&A] [answered]
This is a useful library. Is there API level access to the WiFi parental controls that are available on the Devolo PLC devices? I've not yet found it if there is.

**Comment by Shutgun** [comment, marked as answer]
Hi @wingwang , currently the API has no support for accessing the parental controls. But I will ask my contacts at devolo, if they can add it and come back to you.

**Reply by wingwang** [reply]
Thanks, that would be great. I suggest that the easiest way of introducing this feature, at least initially, would be to add enabling and disabling the parental control feature. Adding and removing devices, and setting up time limits, is less important and could come later, if at all.
